.github/workflows/go.yml

```diff
@@ -0,0 +1,34 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+    strategy:
+      matrix: 
+        go-version: [1.12.x, 1.13.x, 1.14.x]
```

Comment:
Just curious, are we also adding Go 1.14.x to the build matrix?

Reply:
@dims showed klog repo which uses 1.14 and kinda seemed best practice to build for 1 up and one down with 13 being the middle as several folks may be around 14 and 15 or 15+ would be bleeding-edge. Should i remove it ? 1.14 was not in the original travis yaml